modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -80,7 +237,14 @@ export class ReceiveVideoController {
         this._maxFrameHeight = maxFrameHeight;
 
         for (const session of this._conference._getMediaSessions()) {
-            maxFrameHeight && session.setReceiverVideoConstraint(maxFrameHeight);
+            if (session.isP2P || !this._receiverVideoConstraints) {
```

Comment:
Is `setReceiverVideoConstraint` actually used in p2p?

Reply:
Yes, JingleSessionPC.setReceiverVideoConstraint sends out a content-modify for requesting a different resolution in p2p mode and uses the legacy format for jvb mode. https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/JingleSessionPC.js#L1311